test/libsolidity/SemanticTest.cpp

```diff
@@ -80,12 +80,19 @@ bool SemanticTest::run(ostream& _stream, string const& _linePrefix, bool _format
 	{
 		AnsiColorized(_stream, _formatted, {BOLD, CYAN}) << _linePrefix << "Expected result:" << endl;
 		for (auto const& test: m_tests)
-			_stream << test.format(_linePrefix, false, _formatted) << endl;
+		{
+			ErrorReporter errorReporter;
+			_stream << test.format(errorReporter, _linePrefix, false, _formatted) << endl;
+			errorReporter.print(_stream, _formatted, _linePrefix);
```

Comment:
```suggestion
			_stream << errorReporter.format(_linePrefix, _formatted);
```
This line will look like that in the end.